### PR TITLE
Remove the static config

### PR DESCRIPTION
**Pull request by dancormier**
This PR removes `lib/css/base/_stacks-configuration-static.less`, and moves necessary Less variables (`@css-reset`, `@breakpoints`) to `lib/css/exports/_stacks-constants-helpers.less`.

~~I've yet to test this against core so standby.~~

I've tested this against core and it LGTM but I'd love some other eyes on it.

---

See also:

PR comment saying "we're coming for you next #stacks-config-static"

### CODE REVIEW

File: lib/css/stacks-static.less [hunk 1]
```diff
@@ -83,4 +83,3 @@
 
 //  --  CONFIG
 @import "base/configuration-static.less";
```
Comment (aaronshekey) [inline lib/css/stacks-static.less]: I think this one's gotta go yet.

Comment (dancormier) [reply lib/css/stacks-static.less]: Ah, damn funky merge. Removed!

Also, thanks for catching this!

**Review by aaronshekey — APPROVED** [review]

**Review by b-kelly — CHANGES_REQUESTED** [review]
Looks like we've lost the ability to set css resets and multiple breakpoints - both of which we're using in prod. I'm going to see if I can come up with a good solution here.

**Comment by dancormier** [comment]
> Looks like we've lost the ability to set css resets and multiple breakpoints - both of which we're using in prod. I'm going to see if I can come up with a good solution here.

Is the reset not handled through `lib/css/base/reset.less`?

```
// the default CSS reset is normalize.css; other possibilities are the Eric Meyer reset, and no reset at all.

#stacks-internals() {
    #include-css-reset(meyer) {
        @import "_stacks-reset-meyer.less";
    }
    #include-css-reset(normalize) {
        @import "_stacks-reset-normalize.less";
    }
    /* stylelint-disable block-no-empty */
    #include-css-reset(none) {
    }
    /* stylelint-enable */
}

& {
    #stacks-internals #load-static-config();
    #stacks-internals #include-css-reset(@css-reset);
}
```

**Comment by b-kelly** [comment]
@dancormier Unfortunately, no. I just double checked and downstream consumers aren't setting this value with the current syntax. Working on a downstream fix now to see if we need to make any changes here.

**Comment by b-kelly** [comment]
I'm starting to think we can't merge this without an overhaul on how we deliver breakpoints and resets. Any concerns on leaving this for 2.0?

**Comment by dancormier** [comment]
> I'm starting to think we can't merge this without an overhaul on how we deliver breakpoints and resets. Any concerns on leaving this for 2.0?

Yep, sounds good.

**Pull request closed without merging**